spring-xd-dirt/src/main/resources/application.yml

```diff
@@ -215,35 +218,44 @@ xd:
         - GET    /jobs/executions/*/steps/*      => hasRole('ROLE_VIEW')
         - GET    /jobs/executions/*/steps/*/progress => hasRole('ROLE_VIEW')
         # Batch Job Instances
-        - GET    /jobs/instances?jobname=*       => hasRole('ROLE_VIEW')
+        - GET    /jobs/instances*                => hasRole('ROLE_VIEW')
```

Comment:
Looks like query strings are stripped when using ant paths in Spring Security. See: http://docs.spring.io/spring-security/site/docs/current/reference/htmlsingle/#request-matching

You can switch to regular expression matching for more complicated matches but I don't know if that level of sophistication is really needed here.